RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -952,6 +953,112 @@ private boolean parseAutopsyPluginOutput(String regFilePath, AbstractFile regFil
         return false;
     }
 
+    
+    private boolean parseSystemPluginOutput(String regfilePath, AbstractFile regAbstractFile) {
+        File regfile = new File(regfilePath);
+        try (BufferedReader reader = new BufferedReader(new FileReader(regfile))) {
+            String line = reader.readLine();
+            while (line != null) {
+                line = line.trim();
```

Comment:
Could this line be made lowercase so that the regexes below match without casing being an issue?

Reply:
Done